<launch>
	<arg name = "prefix" default = "new_bags/subject3_6" />
	<rosparam file="$(find gait_hmm_ros)/scripts/$(arg param_file)" command="load"/>
	<arg name = "matfile" default = "none" />
	<node name = "bag2mat" pkg="gait_hmm_ros" type = "bag2mat.py" output = "screen" required = "true" >
		<param name = "prefix" value = "$(arg prefix)" />
		<param name = "matfile" type = "" value = "$(arg matfile)" />
		<param name = "rf" type = "string" value = "" />
		<param name = "rll" type = "string" value = "" />
		<param name = "rul" type = "string" value = "" />
		<param name = "lf" type = "string" value = "" />
		<param name = "lll" type = "string" value = "" />
		<param name = "lul" type = "string" value = "" />
		<param name = "m" type = "string" value = "" />
		<param name = "ch" type = "string" value = "" />
		<param name = "h" type = "string" value = "" />
		<param name = "ls" type = "string" value = "" />
		<param name = "rs" type = "string" value = "" />
		<param name = "rua" type = "string" value = "" />
		<param name = "rla" type = "string" value = "" />
		<param name = "rh" type = "string" value = "" />
		<param name = "lua" type = "string" value = "" />
		<param name = "lla" type = "string" value = "" />
		<param name = "lh" type = "string" value = "" />
	</node>
</launch>
